api/cluster/src/main/java/org/pentaho/big/data/api/cluster/NamedCluster.java

```diff
@@ -35,6 +35,10 @@
 
   void setName( String name );
 
+  String getShimIdentifier();
+
+  void setShimIdentifier( String shimIdentifier );
+
```

Comment:
Why do we need a shim identifier property in the Named Cluster?  Won't hdi just be identified in the same way as any other shim-- as the active configuration?